File: Basic/Hello World/Java/test/com/hpclatam/helloTest.java
package com.hpclatam;

import static org.junit.Assert.*;

import org.junit.*;

public class helloTest {

    public helloTest(){

    } 

	@Test
	public void testSayHello() {
		Hello hello = new Hello();
		assertEquals("Hello World", hello.sayHello());
	}

}

File: Basic/Hello World/PHP/tests/HelloTest.php
<?php
declare(strict_types=1);

use PHPUnit\Framework\TestCase;


final class Hello
{
  private function __construct()
    {
    }

    public function sayHello(): string
    {
        return 'Hello World';
    }


}

final class HelloTest extends TestCase
{

    public function testDisplayHelloWorld()
    {
        $this->assertEquals(
            'Hello World',
            Hello::sayHello()
        );
    }
}
?>